core/src/main/scala/org/ensime/core/javac/Helpers.scala

```diff
@@ -47,13 +47,24 @@ trait Helpers extends UnsafeHelpers with SLF4JLogging {
     Some(JavaFqn(front.mkString("."), back.mkString("."), None))
   }
 
-  private val isSimpleKind = Set(ElementKind.LOCAL_VARIABLE, ElementKind.PARAMETER)
-  private val isComplexKind = Set(ElementKind.CONSTRUCTOR, ElementKind.ENUM_CONSTANT, ElementKind.METHOD, ElementKind.FIELD)
+  private class isOneOf(options: ElementKind*) {
```

Comment:
pattern matchers are typically uppercased.

That said, since this is basically a switch statement, we could just do

`case LOCAL_VARIABLE | Parameter | ... => `

and avoid any intermediates.